dbms/src/Storages/DeltaMerge/ReadThread/CPU.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 PingCAP, Ltd.
```

Comment:
Should we remove this file from this pr?

Reply:
removed